maintainers.txt

```diff
@@ -0,0 +1,3 @@
+lsh-0
```

Comment:
Think at the moment luke is the only maintainer of spectrum, so might be best to drop nathan and graham.